### ISSUE TITLE
No jmx user/password set during Kafka service creation

### ISSUE THREAD

**Issue by jazzl0ver** [cloudstax/firecamp#59]
```
$ PREFIX=0.9.6
$ $PREFIX/firecamp-service-cli -op=create-service -service-type=kafka -region=us-east-1 -cluster=firecamp-$ENV -replicas=3 -volume-size=$VOLSZ -service-name=kafka-$ENV -kafka-zk-service=zoo-$ENV -kafka-heap-size=512
The Kafka heap size is less than 6144. Please increase it for production system
2018-05-18 11:00:12.399656608 +0000 UTC The kafka service is created, jmx user  password
...
```

**Comment by JuniusLuo** [comment]
Thanks for reporting it! The jmx user and passwd were actually created, but somehow not pass back to cli. We will fix it in the next release. For now, you could use get-service to get the kafka jmx user and passwd. firecamp-service-cli -op=get-service -cluster=firecamp-$ENV -service-name=kafka-$ENV.

**Comment by cloudstax** [comment]
This was fixed in 1.0.
